code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -4,6 +4,7 @@
 	icon = 'icons/obj/guns/ammo.dmi'
 	icon_state = "s-casing"
 	worn_icon_state = "bullet"
+	note_override = 1
```

Comment:
And notes_override doesn't need to exist, since you can just, not call the parent proc